lib/puppet/bindings.rb

```diff
@@ -129,7 +129,7 @@ def [](name)
     end
 
     def has_name?(name)
-      @named_bindings.has_key?
+      @named_bindings.has_key?(name)
```

Comment:
Even if this certainly is wrong, I would prefer if it was left as is. Reason being that it's not used and we are removing the binder on master. Fixing this will create conflicts down the line.

Reply:
dead as a doornail - file should be removed

Reply:
See https://github.com/puppetlabs/puppet/pull/6405